modules/clydoBidAdapter.js

```diff
@@ -0,0 +1,139 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { deepSetValue, deepAccess, isFn } from '../src/utils.js';
+import { toOrtbNativeRequest } from '../src/native.js';
+import { BANNER, VIDEO, NATIVE } from '../src/mediaTypes.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+
+const BIDDER_CODE = 'clydo';
+const METHOD = 'POST';
+const params = {
+  region: "{{region}}",
+  partnerId: "{{partnerId}}"
+}
+const BASE_ENDPOINT_URL = `https://${params.region}.clydo.io/${params.partnerId}`
+
+const converter = ortbConverter({
+  context: {
+    netRevenue: true,
+    ttl: 30
+  }
+});
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
+  userSync: {
+    topics: false,
+  },
```

Comment:
This isn't a real field in the overall spec.  There is a field called `getUserSyncs` that lets you execute a userSync URL, but it doens't influence things with topics.